### PR TITLE
Adds a new bump version func to the API.

### PR DESCRIPTION
**Pull request by RickEyre**
For #16.

### CODE REVIEW

**Comment by rbarrois** [comment]
Hi,

Thanks for the proposal :)

The features looks great; however, since ``Version`` objects are hashable, they have to stay immutable.

Thus, I'd rather use the following API, where each function returns a *new* object:

```python
>>> v = semantic_version.Version('1.2.3-rc2')
>>> v.next_patch()
Version('1.2.4')
>>> v.next_minor()
Version('1.3.0')
>>> v.next_major()
Version('2.0.0')
```

Could you update your PR with this?
Also, could you add your name to the contributors list (https://github.com/rbarrois/python-semanticversion/blob/master/CREDITS#L15)?

**Comment by RickEyre** [comment]
Thanks for the quick feedback ! :smile: 

Just to clarify, would you rather have `next_major`, `next_minor`, `next_patch` or `bump` which takes one of `'minor', 'major', 'patch'`.

**Comment by RickEyre** [comment]
@rbarrois updated the PR, just went with what you suggested :smile:

**Comment by RickEyre** [comment]
@rbarrois when do you think we could merge this?

**Comment by IanLee1521** [comment]
:+1:

File: README.rst [hunk 1]
```diff
@@ -112,6 +112,22 @@ Obviously, :class:`Versions <Version>` can be compared:
...
+.. code-block:: pycon
+
+    >>> v = semantic_version.Version('0.1.1-pre+build')
+    >>> new_v = v.next_major()
+    >>> str(new_v)
+    '1.0.0'
+    >>> v = semantic_version.Version('1.1.1-pre+build')
+    >>> new_v = v.next_minor()
+    >>> str(new_v)
+    '1.1.0'
```
Comment (IanLee1521) [inline README.rst]: Shouldn't the new version be '1.2.0' ?

Comment (RickEyre) [inline README.rst]: Correct, I actually hand wrote this instead of using the repl.. I'll update.

**Comment by rbarrois** [comment]
Merged as 0b4d4c724f6d86bbc2c964af0463ef28de7c8015.

**Pull request closed without merging**